specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/SourceControls.json

```diff
@@ -360,6 +360,10 @@
             "name"
           ],
           "description": "List of environment variables to be passed to the build."
+        },
+        "label": {
```

Comment:
I'm not sure if this is captured but label should be optional right?  And if they're in labeled revision mode and forget to include it, I assume it should throw?

Reply:
Yes, this is optional and the API will do the validation.